src/views/Home/components/CLWCreditsDaysLeft/index.jsx

```diff
@@ -59,7 +59,7 @@ const CLWCreditsDaysLeft = () => {
         // Allow different tooltips for different datasets within the same pie;
         callbacks: {
           // Code taken from https://github.com/chartjs/Chart.js/issues/1417
-          label: function(item, data) {
+          label: function (item, data) {
```

Comment:
```suggestion
          label: function(item, data) {
```
The extra space looks like it might have been accidental.